extensions/src/AWSSDK.Extensions.Configuration.AWSSystemsManager/AWSSDK.Extensions.Configuration.AWSSystemsManager.nuspec

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata>
+    <id>AWSSDK.Extensions.Configuration.AWSSystemsManager</id>
+    <title>AWSSDK - ASP.NET Configuration Extensions for AWS Systems Manager</title>
+    <version>1.0.0</version>
+    <authors>Amazon Web Services, Ken Hundley</authors>
+    <description>ASP.NET Configuration Extensions for AWS Systems Manager</description>
+    <language>en-US</language>
+    <licenseUrl>http://aws.amazon.com/apache2.0/</licenseUrl>
+    <projectUrl>https://github.com/aws/aws-sdk-net/</projectUrl>
```

Comment:
once we create a new repo, we should probably update this.  I'll take a note so we don't forget.

Reply:
Per our conversation, we should probably drop ASP.NET from the description :)

Reply:
Agreed on the ASP.Net. What would you like instead?

"Host Configuration Extensions for AWS Systems Manager"?

Reply:
perfect